adminer-editor/Dockerfile

```diff
@@ -17,15 +17,16 @@ RUN echo '@testing http://nl.alpinelinux.org/alpine/edge/testing' >> /etc/apk/re
         php7-session@testing \
         php7-mysqli@community \
         php7-pgsql@testing \
-        php7-pecl-mongodb@testing && \
+        php7-pecl-mongodb@testing \
+        dumb-init && \
     wget https://github.com/vrana/adminer/releases/download/v$ADMINER_EDITOR_VERSION/editor-$ADMINER_EDITOR_VERSION.php -O /srv/index.php && \
     apk del wget ca-certificates && \
     rm -rf /var/cache/apk/*
 
 WORKDIR /srv
 EXPOSE 80
 
-CMD /usr/bin/php \
+CMD dumb-init -- /usr/bin/php \
```

Comment:
According to [this](https://www.ctl.io/developers/blog/post/dockerfile-entrypoint-vs-cmd/) you should always use the exec form. When I implemented it in my other project I did this:

```
ENTRYPOINT ["dumb-init", "--"]

CMD [...]
```

Reply:
This way the image will always use dumb-init as PID 1 (unless user overrides it with `--entrypoint`) even if the user starts the image with a different command.

Reply:
done, thanks